### DISCUSSION TITLE
Cannot init - "cannot create .gitignore"

### DISCUSSION THREAD

**Discussion by EinfachNurBaum** [PyO3/maturin#1413] [Q&A]
### Bug Description

I can run the init command and choose pyo3 but then I get the errror "cannot create .gitignore"
![image](https://user-images.githubusercontent.com/82251642/212472034-8d395c8c-a440-42bb-99dd-af4008bb87f2.png)

### Your maturin version (`maturin --version`)

0.14.1

### Your Python version (`python -V`)

Python 3.10

### Your pip version (`pip -V`)

22.3

### What bindings you're using

None

### Does `cargo build` work?

- [X] Yes, it works

### If on windows, have you checked that you aren't accidentally using unix path (those with the forward slash `/`)?

- [ ] Yes

### Steps to Reproduce

Just installed maturin with pip and tried to run "maturin init" it

**Comment by messense** [comment]
Please check your file system permission, it's running fine in CI: https://github.com/PyO3/maturin/actions/runs/3908040881/jobs/6677854390#step:20:21 

(`maturin new` is basically the same as `mkdir` + `maturin init`.)

**Reply by messense** [reply]
See also https://github.com/flutter/flutter/issues/114609#issuecomment-1303068612

> The issue was caused by windows defender as it blocked the access to the file system.
